_test_yaml/test/src/build_config.g.dart

```diff
@@ -103,28 +103,22 @@ Builder _$BuilderFromJson(Map json) => $checkedCreate(
       },
     );
 
-Map<String, dynamic> _$BuilderToJson(Builder instance) {
-  final val = <String, dynamic>{};
-
-  void writeNotNull(String key, dynamic value) {
-    if (value != null) {
-      val[key] = value;
-    }
-  }
-
-  writeNotNull('target', instance.target);
-  writeNotNull('import', instance.import);
-  writeNotNull('is_optional', instance.isOptional);
-  writeNotNull('configLocation', instance.configLocation?.toString());
-  writeNotNull('auto_apply', _$AutoApplyEnumMap[instance.autoApply]);
-  writeNotNull('build_to', _$BuildToEnumMap[instance.buildTo]);
-  writeNotNull('defaultEnumTest', _$AutoApplyEnumMap[instance.defaultEnumTest]);
-  val['builder_factories'] = instance.builderFactories;
-  writeNotNull('applies_builders', instance.appliesBuilders);
-  writeNotNull('required_inputs', instance.requiredInputs);
-  writeNotNull('build_extensions', instance.buildExtensions);
-  return val;
-}
+Map<String, dynamic> _$BuilderToJson(Builder instance) => <String, dynamic>{
+      if (instance.target case final val?) 'target': val,
```

Comment:
Any reason not to use a proper word like `value`?

Reply:
yep